README-PLAYBOOK.md

```diff
@@ -1,164 +1,164 @@
-| jksPassword          | string  | Required when type is `JKS`                                                                                                                                                                                    |
-| location             | string  | file location for certificate. (NOTE: When type is `PEM`, this must be a folder location!)                                                                                                                     |
-| pemCertFilename      | string  | Required when type is `PEM`                                                                                                                                                                                    |
-| pemChainFilename     | string  | Required when type is `PEM`                                                                                                                                                                                    |
-| pemKeyFilename       | string  | Required when type is `PEM`                                                                                                                                                                                    |
-| type                 | string  | Valid types are `PKCS12`, `PEM`, `JKS`, and `CAPI`                                                                                                                                                             |
+| Field                | Type    | Description |
+|----------------------|---------|-------------|
+| format               | string  |  ***REQUIRED*** - Specifies the format type for the installed certificate. Valid types are `PKCS12`, `PEM`, `JKS`, and `CAPI`. |
+| afterInstallAction   | string  | *OPTIONAL* - Execute this command after this installation is performed (both enrollment and renewal). On *nix, this uses `/bin/sh -c '<afterInstallAction>'`. On Windows, this uses `powershell.exe '<afterInstallAction>'`. |
+| backupFiles          | boolean | *OPTIONAL* - When `true`, backup existing certificate files before replacing during a renewal operation. Defaults to `false`. |
+| file                 | string  | ***REQUIRED*** when [Installation.format](#installation) is `PKCS#12`, `PEM`, or `JKS`. Specifies the file path and name for the certificate file (PEM) or PKCS#12 / JKS bundle. Example `/etc/ssl/certs/myPEMfile.cer`, `/etc/ssl/certs/myPKCS12.p12`, or `/etc/ssl/certs/myJKS.jks`. *IGNORED* for other values of [Installation.format](#installation). |
+| jksAlias             | string  | ***REQUIRED*** when [Installation.format](#installation) is `JKS`. Specifies the certificate alias value within the Java Keystore. *IGNORED* when [Installation.format](#installation) is not `JKS`. |
+| jksPassword          | string  | ***REQUIRED*** when [Installation.format](#installation) is `JKS`. Specifies the password for the JKS. *IGNORED* when [Installation.format](#installation) is not `JKS`. |
```

Comment:
Change JKS acronym here for Java Keystore to be consistent with jksAlias' description:
`Specifies the password for the Java Keystore`